table/src/test/java/com/bazaarvoice/emodb/table/generic/BlockFileTableSetTest.java

```diff
@@ -198,6 +198,7 @@ public void testUnknownAndDroppedTable() throws Exception {
 
     @Test
     public void testDistributed() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS00112)